pkg/cpi/mockingUtils.go

```diff
@@ -41,3 +56,27 @@ func GetCPIFunctionMockResponse(functionName, testType string) (*http.Response,
 		return &res, errors.New("Service not Found")
 	}
 }
+
+//GetEmptyHTTPResponseBody -Empty http respose body
+func GetEmptyHTTPResponseBody() (*http.Response, error) {
+	res := http.Response{
+		StatusCode: 202,
+		Body:       ioutil.NopCloser(bytes.NewReader([]byte(``))),
+	}
+	return &res, nil
```

Comment:
Why return an error if it is always `nil`?

Reply:
this i am doing to have test coverage of all lines of the step when response status is not 202 but other codes such as 400 or 500 etc

Reply:
But `GetEmptyHTTPResponseBody` does not need to return an error. I would suggest adding the nil-error to the return statement where `GetEmptyHTTPResponseBody` is used.